lib/ProductOpener/Images.pm

```diff
@@ -428,6 +428,42 @@ sub dims {
 }
 
 
+
+=head2 get_code_and_imagefield_from_file_name ( $l $filename )
+
+This function is used to guess if an image is the front of the product,
+its list of ingredients, or the nutrition facts table, based on the filename.
+
+It is used in particular for bulk upload of photos sent by manufacturers.
+The file names have many different formats, but they very often include the barcode of the product,
+and sometimes an indication of what the image is.
+
+Producers are advised to use the [front|ingredients|nutrition|packaging]_[language code] format,
+but in practice we receive many other names.
+
+The %file_names_to_imagefield_regexps structure below contains some patterns
+used to guess what the image is about.
+
+=cut
+
+# the regexps apply to canonicalized strings
+# i.e. lowercased / unaccented strings in most European languages
+my %file_names_to_imagefield_regexps = (
+
+	en => [
+		["ingredients" => "ingredients"],
+		["nutrition" => "nutrition"],
+	],
+	es => [
+		["ingredientes" => "ingredients"],
+		["nutricion" => "nutrition"],	
+	],
+	fr => [
+		["ingredients" => "ingredients"],
+		["nutrition" => "nutrition"],
+	],
```

Comment:
@hangy @VaiTon Could you add Italian and German equivalent (ingredienti, Zutaten…)